### ISSUE TITLE
bug in pefile ?

### ISSUE THREAD

**Issue by joelpx** [mlaferrera/python3-pefile#1]
This issue is just for your information if a similar bug occurs.

I got sometimes a bug in my project when I used the repo https://github.com/simonzack/pefile-py3k , but when I updated to yours, I have not been able to reproduce the bug.

However I think there is still an undefined behavior, you can check a look on my issue here : joelpx/reverse#16

**Comment by mlaferrera** [comment]
I ran into issues with that repository as well..not everything in it was ported over to a proper implementation for python3. 

Thanks for the heads up on the odd behavior. Perhaps it would be a good idea to file this bug with Ero's pefile github page as well? I want to avoid making any modifications to this library aside from what is necessary for python3 support. Keeping it as close as possible to the primary library will make results consistent between the two.

**Comment by joelpx** [comment]
Yes why not, it could be a good idea to add a discussion on the official page. But for the moment with the few tests I have, it's ok. Maybe by pure luck.

Is your project became the official port of pefile for python3 ?

**Comment by mlaferrera** [comment]
No, definitely not the official. I'm not even sure Ero know's of the existence of this mirror.
